test/Sign.Core.Test/SignatureProviders/DistinguishedNameParserTests.cs

```diff
@@ -0,0 +1,41 @@
+namespace Sign.Core.Test
+{
+    public class DistinguishedNameParserTests
+    {
+        [Fact]
+        public void Parse_WhenSubjectIsValid_ReturnsRelativeDistinguishedNames()
+        {
+            string subject = "CN=Microsoft Code Signing PCA 2011, O=Microsoft Corporation, L=Redmond, S=Washington, C=US";
```

Comment:
We should have a test for a subject name that contains internal (escaped) quotes. Many parsers fail at that.

Reply:
https://github.com/dotnet/sign/issues/451